site/reference/cli/alpha/rpkg/copy/README.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "`copy`"
+linkTitle: "copy"
+type: docs
+description: >
+  Create a new package revision from an existing one.
+---
+
+<!--mdtogo:Short
+    Create a new package revision from an existing one.
+-->
+
+`copy` creates a new package revision from an existing one. The new
+revision will be identical to the existing one but with a different
+name.
```

Comment:
`name or revision` ?

(since we are considering the default behavior to keep name the same and only rev the revision)

Reply:
Agree revision is a better word here. Updated.